### PR TITLE
vips: update to 8.14.2

### PR DESCRIPTION
**Pull request by flyn-org**
Maintainer: me
Compile tested: x86_64

### CODE REVIEW

**Comment by 1715173329** [comment]
```
Checking for function "ngettext" : YES 
Checking for function "bind_textdomain_codeset" : YES 
Configuring config.h using configuration
Program glib-mkenums mkenums found: NO

../../../builder/build_dir/target-mips_24kc_musl/vips-8.14.2/libvips/include/vips/meson.build:60:0: ERROR: Program 'glib-mkenums mkenums' not found or not executable

A full log can be found at /builder/build_dir/target-mips_24kc_musl/vips-8.14.2/openwrt-build/meson-logs/meson-log.txt
make[2]: *** [Makefile:92: /builder/build_dir/target-mips_24kc_musl/vips-8.14.2/.configured_a17fb5ef857664f03cd0ce37cc5ea591] Error 1
make[2]: Leaving directory '/feed/libs/vips'
time: package/feeds/packages_ci/vips/compile#2.51#0.71#2.97
    ERROR: package/feeds/packages_ci/vips failed to build.
```

**Comment by neheb** [comment]
Sounds like host glib2 is needed as a build dependency

File: libs/vips/Makefile [hunk 1]
```diff
@@ -21,9 +21,11 @@ PKG_CPE_ID:=cpe:/a:vips:vips
 PKG_FIXUP:=autoreconf
```
Comment (neheb) [inline libs/vips/Makefile:21]: Both not needed with meson

Comment (neheb) [reply libs/vips/Makefile:21]: Actually all three. Parallel, install, and fix up.

File: libs/vips/Makefile [hunk 2]
```diff
@@ -34,38 +36,35 @@ define Package/vips
   DEPENDS:=+glib2 +libexif +libexpat +libjpeg +libpng +giflib +libxml2 +libstdcpp $(INTL_DEPENDS)
```
Comment (neheb) [inline libs/vips/Makefile:36]: Wonder if libstdcpp is needed. intl_depends comes with glib2.

Comment (flyn-org) [reply libs/vips/Makefile]: Removed both $(INTL_DEPENDS) and +libstdcpp.

File: libs/vips/Makefile (hunk 1 above)
Comment (flyn-org) [reply libs/vips/Makefile]: Removed.

Comment (flyn-org) [reply libs/vips/Makefile]: Removed PKG_INSTALL.

**Pull request merged**